go/src/koding/klient/Dockerfile

```diff
@@ -6,5 +6,5 @@ ENV GOPATH /opt/koding/go
 WORKDIR /opt/koding
 
 RUN apt-get update && apt-get install -y git build-essential ubuntu-dev-tools dh-make && apt-get clean && \
-    curl -sSL https://storage.googleapis.com/golang/go1.7.4.linux-amd64.tar.gz | tar -C /usr/local/ -zxf - && \
+    curl -sSL https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz | tar -C /usr/local/ -zxf - && \
```

Comment:
could we remove this dockerfile and use koding's base docker image? lets not maintain two images

cc/ @szkl